### PR TITLE
Created the Intake Subsystem

### PR DESCRIPTION
**Pull request by YJ-7274**

### CODE REVIEW

**Comment by prensing** [comment]
Looks fine.

**Review by prensing — APPROVED** [review]
YJ, please use the "m_" for the class variables

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Intake extends SubsystemBase {
+    
+    CANSparkMax intakeMotor;
```
Comment (prensing) [inline src/main/java/frc/robot/subsystems/Intake.java:11]: This is a member variable, should be "m_"

**Review by prensing — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    @Override
+    public void periodic() {
+        // This method will be called once per scheduler run
+    }
+
+    public void run(double speed) {
+        intakeMotor.set(-speed);
+    }
+
+    public void IntakeCargo() {
```
Comment (prensing) [inline src/main/java/frc/robot/subsystems/Intake.java:26]: Method start with a lowercase letter.

**Review by prensing — CHANGES_REQUESTED** [review]

**Review by prensing — APPROVED** [review]
Looks good now.

**Review by coachFitz0807 — APPROVED** [review]

**Pull request merged**